qiskit_experiments/test/pulse_backend.py

```diff
@@ -395,18 +421,28 @@ def run(self, run_input: Union[QuantumCircuit, List[QuantumCircuit]], **run_opti
             state_t = self.ground_state.copy()
             for instruction in circuit.data:
                 qubits, params, inst_name = self._get_info(circuit, instruction)
-                if inst_name in ["barrier", "measure"]:
+                if inst_name == "barrier":
+                    continue
+                if inst_name == "measure":
+                    meas_qubits += [qubits[0]]
                     continue
                 if inst_name == "rz":
+                    # TODO: Fix: Assumes 3 level system.
```

Comment:
Are you assuming three-level for the two-qubit backend? Two level may be easier to start of with.

Reply:
Oh, well, the single and parallel transmon backends have three levels, I just made this comment as a reminder to change it, so it also supports the 2 level 2 qubit backend. Do you think we should move the Rz definition inside the subclass and document it so that the base class is cleaner?